### PR TITLE
Feature/consoleCreateProject

### PR DESCRIPTION
**Pull request by GuentherJulian**

### CODE REVIEW

File: runners/console/index.ts [hunk 1]
```diff
@@ -60,6 +60,20 @@ export class Console extends Runner {
         return result;
     }
 
+    runCreateProject(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
```
Comment (maybeec) [inline runners/console/index.ts:66]: there is no differentiation on command.parameters[0].
I think we should throw an exception somehow to indicate if a template was used, which does not exist for easier debugging and understanding of failures

File: documentation/Functions.md [hunk 2]
```diff
@@ -1,3 +1,44 @@
...
+
+***
+
+### createProject
+#### parameter 
+1. the programming language that you will use for this project
+2. the project name
+
+#### example 
+createProject("java","cobigenexample")
```
Comment (maybeec) [inline documentation/Functions.md:42]: let's not focus on the programming language as a first parameter as there might be different architectures for one programming language bound to different project templates in near future.
So let's focus on a parameter like devon4j to indicate devon4j project template.

File: runners/console/index.ts [hunk 3]
```diff
@@ -60,6 +60,20 @@ export class Console extends Runner {
...
+        let workspaceDir = path.join(this.getWorkingDirectory(), "devonfw", "workspaces", "main")
+        if(this.platform == ConsolePlatform.WINDOWS) {
+            this.executeCommandSync("devon " + command.parameters[0] + " create com.example.application." + command.parameters[1], workspaceDir, result);
```
Comment (maybeec) [inline runners/console/index.ts:69]: where is the second parameter here taken into account?

**Review by maybeec — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 2 above)
Comment (GuentherJulian) [reply documentation/Functions.md]: Do you mean a differentiation between devon4j, devon4net, etc?
Should we create separate tasks for these?

Comment (maybeec) [reply documentation/Functions.md]: Yes and even within devon4j there might be more than one option to create a project.

I am not sure, possibly it's more simpler to have a task per stack like devon4j vs devon4ng.

File: documentation/Functions.md [hunk 4]
```diff
@@ -35,10 +35,10 @@ cobiGenJava("path/to/java/file/MyEntity.java",[1,3,5,6,8])
 
 ### createProject
 #### parameter 
-1. the programming language that you will use for this project
+1. the template used for project generation
 2. the project name
 
 #### example 
-createProject("java","cobigenexample")
+createProject("devon4j","cobigenexample")
```
Comment (maybeec) [inline documentation/Functions.md:42]: I even thought of creating a step for it as there will be no logic at all to be reused. So we should split this into createDevon4jProject("cobigenexample") and further

**Review by maybeec — CHANGES_REQUESTED** [review]

File: runners/console/index.ts [hunk 5]
```diff
@@ -92,6 +107,19 @@ export class Console extends Runner {
         console.log("assertCobiGenJava");
```
Comment (maybeec) [inline runners/console/index.ts:107]: let's properly console log that there is no assertion at all. 
We most properly need to address it later

**Review by maybeec — CHANGES_REQUESTED** [review]

**Review by maybeec — APPROVED** [review]

**Pull request merged**